packages/core/test/lib/request.test.ts

```diff
@@ -40,11 +40,12 @@ describe('eventToSentryRequest', () => {
       transaction: '/dogs/are/great/',
       type: 'transaction',
       user: { id: '1121', username: 'CharlieDog', ip_address: '11.21.20.12' },
+      sdkProcessingMetadata: {},
```

Comment:
`sdkProcessingMetadata` property in `Event` is [optional](https://github.com/getsentry/sentry-javascript/blob/afd10544de537e6a8933c61221277de5917c5e55/packages/types/src/event.ts#L48). `eventToSentryRequest` is expecting an event, and all test cases assume the property to always be defined. There should be a test ensuring `eventToSentryRequest` behaves accordingly when `sdkProcessingMetadata` isn't defined.

Reply:
All event fields are optional, actually. I'll add this test in a separate PR:

```javascript
it("doesn't depend on optional event fields for success ", () => {
  // all event fields are optional
  const emptyEvent = {};

  const result = eventToSentryRequest(emptyEvent, api);

  expect(result).toEqual({
    // The body isn't empty because SDK info gets added in `eventToSentryRequest`. (The specifics 
    // of what gets added are tested elsewhere.)
    body: expect.any(String),
    type: 'event',
    url: storeUrl
  });
});
```